OpenMTR/code/Welcome.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenMTR
+{
+    public class Welcome
+    {
+        /// <summary>
+        /// Welcomes the user to the application and gets the directory to process from the console. 
+        /// </summary>
+        public static void WelcomeUser()
+        {
+            // Welcome the user!
+            Console.WriteLine(@"
+----------------------------------
+        Welcome to OpenMTR
+----------------------------------
+You can exit by entering q or quit
+
+
+
+");
+            // Get the directory to process
+            GetProcessDirectory();
+        }
+
+
+        /// <summary>
+        /// Gets and sets the 
+        /// </summary>
+        protected static void GetProcessDirectory()
+        {
+            // Used to get the path from the user, and confirm it exists
+            bool validatingPath = true;
+            string tempPath = "";
+
+            // Get the path and confirm it exists
+            while (validatingPath)
```

Comment:
I feel like we could bail early here by doing something like this:

```c#
Console.Write("Please enter the full path to a folder to process: ");
tempPath = Console.ReadLine();
while(!Directory.Exists(tempPath)){
    if(tempPath == "q" || tempPath == "quit)
    {
         Environment.Exit(0); 
    }
    Console.Write("Directory does not exist. Please enter the full path to a folder to process: ");
    tempPath = Console.ReadLine();
}
Program.DirectoryPath = tempPath;
```

This removes the need for a boolean to keep track of a valid answer and we can recursively ask for user input if the input is invalid